docs/tutorial/index.rst

```diff
@@ -1,20 +1,228 @@
-Tutorial
-========
+Get started with Testflinger CLI
+=================================
 
-Testflinger is a complex piece of software.
+Testflinger is a system for orchestrating the time-sharing of access to a pool of target machines. You can use Testflinger to perform remote real-world test jobs on hardware with optimised utilisation of resources.
 
-This tutorial will introduce you to key concepts, tools, processes and
-operations, starting from your first installation to a cloud deployment.
-Along the way it will give examples of good practice, and pointers to much
-more detailed information.
+Each Testflinger system consists of:
 
-You can expect to spend three to four hours working through the complete
-tutorial. It’s a strongly-recommended investment of time if you’re new to
-Testflinger - it will save you many more hours later on. Follow the
-tutorial steps in sequence; they take you on a learning journey through the
-product.
+* **Testflinger Server**: a microservice that provides APIs and Web UI for dispatching your test jobs in the appropriate queues.
+* **Testflinger Agent**: a per-machine management tool that retrieves jobs from associated queues on the server and processes the jobs.
+* **Device Connector**: a device-specific tool that handles provisioning and other device-specific details for each type of device.
 
-The tutorial has been tested with a variety of users. We make every effort to
-keep it up-to-date and ensure that it’s reliable - but if you encounter any
-problems, we want to help you, so please let us know - `get support <#>`_.
+The **Testflinger CLI** is a command-line interface that facilitates interactions with a Testflinger Server through integrated APIs. You can use Testflinger CLI to submit test jobs, check job status, and retrieve testing results. 
 
+In this tutorial, you will learn how to set up Testflinger CLI, write your first test job, and leverage the CLI to run the test job on a remote Testflinger server. 
+
+
+Prerequisite
+--------------
+
+- Python installed on your system
+- Access to a Testflinger server and a device under test (DUT)
+
+
+Install Testflinger CLI
+--------------------------
+
+The most convenient way to install the CLI is through the *Snap Store*. Open a terminal and run the following command:
+
+.. code-block:: shell
+  
+  $ sudo snap install testflinger-cli
+
+Once the installation is finished, you can execute the ``testflinger-cli`` command to check the installation. For example, you can run with the ``--help`` or ``-h`` option to display the CLI help page, which is useful anytime when you need a reference about how to use the tool:
+
+.. code-block:: shell
```

Comment:
Can we remove the syntax highlighting from this section? It's highlighting some strange words in the preview build.